From: Adam Williamson <awilliam@redhat.com>
Date: Mon, 9 Dec 2013 21:58:34 -0800
Subject: drop several entries from kbd-model-map whose kbd layouts do not
 exist

kbd-model-map was generated from system-config-keyboard's keyboard_models.py.
Several of the kbd layouts referred in that file do not exist and, so far as I
can tell, never did. I believe these entries existed simply to provide the xkb
configuration information for those layouts, and there never were matching kbd
entries; the kbd names were entirely notional, to satisfy the need for some
entry or other in that field.

For systemd, the only function of kbd-model-map is to 'match' kbd and xkb
configurations, so it does not make any sense to maintain entries for cases
where only one or the other exists in this context.

(cherry picked from commit 70dc36d17fbd81eead01fb73cc231c83c6d977e9)
---
 src/locale/kbd-model-map | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/src/locale/kbd-model-map b/src/locale/kbd-model-map
index 1fe9bca..78c7887 100644
--- a/src/locale/kbd-model-map
+++ b/src/locale/kbd-model-map
@@ -4,22 +4,18 @@ sg			ch	pc105		de_nodeadkeys	terminate:ctrl_alt_bksp
 nl			nl	pc105		-		terminate:ctrl_alt_bksp
 mk-utf			mk,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 trq			tr	pc105		-		terminate:ctrl_alt_bksp
-guj			in,us	pc105		guj		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 uk			gb	pc105		-		terminate:ctrl_alt_bksp
 is-latin1		is	pc105		-		terminate:ctrl_alt_bksp
 de			de	pc105		-		terminate:ctrl_alt_bksp
-gur			gur,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 la-latin1		latam	pc105		-		terminate:ctrl_alt_bksp
 us			us	pc105+inet	-		terminate:ctrl_alt_bksp
 ko			kr	pc105		-		terminate:ctrl_alt_bksp
 ro-std			ro	pc105		std		terminate:ctrl_alt_bksp
 de-latin1		de	pc105		-		terminate:ctrl_alt_bksp
-tml-inscript		in,us	pc105		tam		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 slovene			si	pc105		-		terminate:ctrl_alt_bksp
 hu101			hu	pc105		qwerty		terminate:ctrl_alt_bksp
 jp106			jp	jp106		-		terminate:ctrl_alt_bksp
 croat			hr	pc105		-		terminate:ctrl_alt_bksp
-ben-probhat		in,us	pc105		ben_probhat	terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 fi-latin1		fi	pc105		-		terminate:ctrl_alt_bksp
 it2			it	pc105		-		terminate:ctrl_alt_bksp
 hu			hu	pc105		-		terminate:ctrl_alt_bksp
@@ -29,7 +25,6 @@ fr_CH			ch	pc105		fr		terminate:ctrl_alt_bksp
 dk-latin1		dk	pc105		-		terminate:ctrl_alt_bksp
 fr			fr	pc105		-		terminate:ctrl_alt_bksp
 it			it	pc105		-		terminate:ctrl_alt_bksp
-tml-uni			in,us	pc105		tam_TAB		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 ua-utf			ua,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 fr-latin1		fr	pc105		-		terminate:ctrl_alt_bksp
 sg-latin1		ch	pc105		de_nodeadkeys	terminate:ctrl_alt_bksp
@@ -39,16 +34,12 @@ fr-pc			fr	pc105		-		terminate:ctrl_alt_bksp
 bg_pho-utf8		bg,us	pc105		,phonetic	terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 it-ibm			it	pc105		-		terminate:ctrl_alt_bksp
 cz-us-qwertz		cz,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
-ar-digits		ara,us	pc105		digits		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 br-abnt2		br	abnt2		-		terminate:ctrl_alt_bksp
 ro			ro	pc105		-		terminate:ctrl_alt_bksp
 us-acentos		us	pc105		intl		terminate:ctrl_alt_bksp
 pt-latin1		pt	pc105		-		terminate:ctrl_alt_bksp
 ro-std-cedilla		ro	pc105		std_cedilla	terminate:ctrl_alt_bksp
 tj			tj	pc105		-		terminate:ctrl_alt_bksp
-ar-qwerty		ara,us	pc105		qwerty		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
-ar-azerty-digits	ara,us	pc105		azerty_digits	terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
-ben			in,us	pc105		ben		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 de-latin1-nodeadkeys	de	pc105		nodeadkeys	terminate:ctrl_alt_bksp
 no			no	pc105		-		terminate:ctrl_alt_bksp
 bg_bds-utf8		bg,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
@@ -60,11 +51,8 @@ pl2			pl	pc105		-		terminate:ctrl_alt_bksp
 es			es	pc105		-		terminate:ctrl_alt_bksp
 ro-cedilla		ro	pc105		cedilla		terminate:ctrl_alt_bksp
 ie			ie	pc105		-		terminate:ctrl_alt_bksp
-ar-azerty		ara,us	pc105		azerty		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
-ar-qwerty-digits	ara,us	pc105		qwerty_digits	terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 et			ee	pc105		-		terminate:ctrl_alt_bksp
 sk-qwerty		sk	pc105		-		terminate:ctrl_alt_bksp,qwerty
-dev			dev,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 fr-latin9		fr	pc105		latin9		terminate:ctrl_alt_bksp
 fr_CH-latin1		ch	pc105		fr		terminate:ctrl_alt_bksp
 cf			ca	pc105		-		terminate:ctrl_alt_bksp
